frontend/src/stores/roles/actions.ts

```diff
@@ -40,7 +40,7 @@ async function removeRole(roleId: string) {
 }
 
 async function addLevel(level: Level, role: Role): Promise<void> {
-  await post<Level[]>('/rolelevel', level)
+  await post<Level[]>('/levels', level)
```

Comment:
Since you'll not have roleId in the domain object in frontend anymore, you can 

1. destructure this object
2. Pass the roleId, using role.id

All of this in the second parameter